### ISSUE TITLE
Error! Failed to find deployment

### ISSUE THREAD

**Issue by AaronMcCloskey** [amondnet/vercel-action#166]
Not sure if this is on the Vercel side or this action.

I had a successful build 4 hours ago, I then tried to redeploy with no changes to my GitHub workflow but have now come across this error

```
Vercel CLI 27.0.1
Fetching deployment "Vercel CLI 27.0.1  (beta) — https://vercel.com/feedback\nhttps://<preview-url>.vercel.app" in mcclosa
Fetching deployment "Vercel CLI 27.0.1  (beta) — https://vercel.com/feedback\nhttps://<preview-url>.vercel.app" in mcclosa
Error! Failed to find deployment "Vercel CLI 27.0.1  (beta) — https://vercel.com/feedback\nhttps://<preview-url>.vercel.app" in mcclosa
Error! Failed to find deployment "Vercel CLI 27.0.1  (beta) — https://vercel.com/feedback\nhttps://<preview-url>.vercel.app" in mcclosa
Error: The process '/opt/hostedtoolcache/node/16.13.0/x64/bin/npx' failed with exit code 1
```

The `https://<preview-url>.vercel.app` does exist and does seem to deploy according to the vercel website but this error exits the my workflow so any further steps do not run.

This is my step to deploy to vercel.

```
- name: Deploy to Vercel
  uses: amondnet/vercel-action@v25
  id: vercel-action
  with:
    github-token: ${{ secrets.GH_TOKEN }}
    vercel-token: ${{ secrets.VERCEL_TOKEN }}
    vercel-org-id: ${{ secrets.VERCEL_ORG_ID }}
    vercel-project-id: ${{ secrets.VERCEL_PROJECT_ID }}
    # if main branch go to --prod environment otherwise nothing for preview environment
    vercel-args: "${{ github.ref == 'refs/heads/master' && '--prod' || '' }}"
    working-directory: ./web
```

**Comment by amondnet** [comment]
`Fetching deployment "Vercel CLI 27.0.1  (beta) — https://vercel.com/feedback\nhttps://<preview-url>.vercel.app" in mcclosa`
This should be `Fetching deployment "https://<preview-url>.vercel.app" in mcclosa`. Try again with vercel <27.0.1.

**Comment by AaronMcCloskey** [comment]
Thanks, downgrading to `27.0.0` seems to have done the trick.

**Comment by PlentyCodeDev** [comment]
Had the same issue, @michalak111 solution fixed for me, but should we leave this issue open for the new versions?

**Comment by michalak111** [comment]
@PlentyCodeDev there is already a new version of vercel cli released and it should work 27.0.2
